tests/framework/CodeInterfaceTests/gold/raven_running_raven_internal_models/testPointSet_dump.csv

```diff
@@ -1,16 +1,16 @@
 loss_factor,DeltaTimeScramToAux,tollerance,DG1recoveryTime,CladTempThreshold
-3.97904821433,0.0,0.000102768564541,720.324489456,507138.363793
-3.97904821433,1000.0,0.000102768564541,720.324489456,507176.4035
-3.97904821433,2000.0,0.000102768564541,720.324489456,507214.443207
-4.14951117603,0.0,9.63150662775e-05,128.124447802,507113.465546
-4.14951117603,1000.0,9.63150662775e-05,128.124447802,507153.134879
-4.14951117603,2000.0,9.63150662775e-05,128.124447802,507192.804212
-3.9482296471,0.0,0.000103102494869,146.755892585,507116.954975
-3.9482296471,1000.0,0.000103102494869,146.755892585,507154.700056
-3.9482296471,2000.0,0.000103102494869,146.755892585,507192.445138
-3.92810601029,0.0,9.86735092321e-05,396.580726233,507126.616443
-3.92810601029,1000.0,9.86735092321e-05,396.580726233,507164.169142
-3.92810601029,2000.0,9.86735092321e-05,396.580726233,507201.721842
-3.91082378303,0.0,9.97152314658e-05,345.56072516,507124.902745
-3.91082378303,1000.0,9.97152314658e-05,345.56072516,507162.290226
-3.91082378303,2000.0,9.97152314658e-05,345.56072516,507199.677708
+3.97904821433,0.0,0.000102768564541,720.324489456,507089.136682
+3.97904821433,1000.0,0.000102768564541,720.324489456,507184.736697
+3.97904821433,2000.0,0.000102768564541,720.324489456,507280.336712
+4.14951117603,0.0,9.63150662775e-05,128.124447802,507032.492545
+4.14951117603,1000.0,9.63150662775e-05,128.124447802,507128.09256
+4.14951117603,2000.0,9.63150662775e-05,128.124447802,507223.692576
+3.9482296471,0.0,0.000103102494869,146.755892585,507034.274649
+3.9482296471,1000.0,0.000103102494869,146.755892585,507129.874665
+3.9482296471,2000.0,0.000103102494869,146.755892585,507225.47468
+3.92810601029,0.0,9.86735092321e-05,396.580726233,507058.17048
+3.92810601029,1000.0,9.86735092321e-05,396.580726233,507153.770495
+3.92810601029,2000.0,9.86735092321e-05,396.580726233,507249.370511
+3.91082378303,0.0,9.97152314658e-05,345.56072516,507053.2904
+3.91082378303,1000.0,9.97152314658e-05,345.56072516,507148.890415
+3.91082378303,2000.0,9.97152314658e-05,345.56072516,507244.49043
```

Comment:
Could you tell me the reason for regolding?

Reply:
The change is in the 5th and higher digit in most cases, and the model is quite sensitive to small changes in precision, so I feel it is close enough as to be the same, given that this is not an analytic test.